### ISSUE TITLE
Refresh split min-width on window resize.

### ISSUE THREAD

**Issue by frankdiox** [nathancahill/Split.js#44]
Hello! First of all, thanks for provinding this utility.
I'm using Split.js in a simple web app where it's very common to open the developer console, thus resizing the window. I need my left pane to keep a minimum width since otherwise the layout breaks.
Looks like there is no `refresh`/ `fitMin` method exposed to be called on `window.onresize`. Perhaps this could be added for another release?

For now I'm manually triggering click events and somehow it works, although it's pretty ugly and not very efficient :sweat_smile: :

```
var refreshSplit = function() {
  var mousedown = new CustomEvent('mousedown');
  var mousemove = new CustomEvent('mousemove', {bubbles: true});
  var mouseup = new CustomEvent('mouseup', {bubbles: true});
  var gutter = document.querySelector('.gutter-horizontal');

  mousemove.clientX = gutter.getBoundingClientRect().left;

  gutter.dispatchEvent(mousedown);
  gutter.dispatchEvent(mousemove);
  gutter.dispatchEvent(mouseup);
};
```

And then throttling that:

```
var lock;
window.onresize = function() {
  clearTimeout(lock);
  lock = setTimeout(refreshSplit, 100);
};
```

**Comment by nathancahill** [comment]
Great idea. I'll add it to the roadmap.

**Comment by jdimeo** [comment]
+1
I'm reading that accomplishing this via timeout could lead to poor performance... how about this:
https://github.com/sdecima/javascript-detect-element-resize

**Comment by nathancahill** [comment]
I think this should be fixed in v1.2.0, please test.

**Comment by frankdiox** [comment]
@nathancahill I got some time to update to 1.2.0 now, but I'm not sure how this is fixed. Is there any new method to call on window resize? Couldn't find anything and still need to do what I posted originally. Anyway, thanks for the updates! Things like `collapse()` are really useful :+1: 

P.S: About the "semi breaking change", I got the same behavior with some extra code, perhaps you could add something like this to the readme for other users having the same problem:

```javascript
  var vw = window.innerWidth; // if it's fullscreen
  var minPerc = Math.ceil((minPx + gutterSize) * 100 / vw);

  ... sizes: [minPerc, 100 - minPerc],
  ... minSize: [minPx, 0],

```
